### DISCUSSION TITLE
[FR] Allow {% cache %} tags to keep track of html included with {% html %} tags as it does with {% js %} and {% css %} tags

### DISCUSSION THREAD

**Discussion by mike-moreau** [craftcms/cms#11811] [Ideas] [answered]
In Craft 4, it's now possible to wrap `{% js %}` and `{% css %}` in `{% cache %}` tags ([from this previous feature request](https://github.com/craftcms/cms/discussions/9987#discussioncomment-2649284)).

Would it be possible to add support for the `{% html %}` tag also?

One use case where this would be handy (at least for me) is outputting html for a modal element that I want included just before the closing body tag. Currently I can't do this from inside a cache tag.

**Comment by brianjhanson** [comment, marked as answer]
Thanks for the suggestion! Just added this for Craft 4.3 ✨ (#11839)

**Reply by mike-moreau** [reply]
Super!! Thanks much.

**Reply by brandonkelly** [reply]
Craft 4.3 was released earlier this week with that change!
